src/triton_cli/__init__.py

```diff
@@ -24,4 +24,4 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-__version__ = "0.0.7"
+__version__ = "0.0.8"
```

Comment:
I want it to be clear from terminal outputs, screenshots, copy/paste, etc. that this is installed from source and not a release to ease future debugging/issues
```suggestion
__version__ = "0.0.8dev"
```

Reply:
Similar to here: https://github.com/triton-inference-server/server/blob/main/TRITON_VERSION